tests/observability/metrics/test_vms_metrics.py

```diff
@@ -37,15 +37,15 @@
     USED,
 )
 from utilities.infra import get_node_selector_dict
+from utilities.monitoring import get_metrics_value
 from utilities.virt import VirtualMachineForTests, fedora_vm_body, running_vm
 
 LOGGER = logging.getLogger(__name__)
 
 
 def get_vm_metric_value(prometheus, metric, vm):
```

Comment:
This is no longer needed, instead we should directly use get_metrics_value() with correct metrics_name param. This is just a wrapper around the get_metrics_value().

Reply:
Removed this function and directly use get_metrics_value()  in TimeoutSampler.